### ISSUE TITLE
Empty folders 

### ISSUE THREAD

**Issue by ConnorDarghaoui** [msys2/msys2.github.io#148]
When the installation is finished, no errors appear, but the "_bin_" and other folders are empty.

**Comment by ZachBacon** [comment]
What do you mean by bin, the /usr/bin folder or the /mingw64/bin etc folders? Because if it's /mingw64 and other environments that's by design so that users can pick and choose what toolkit they want using pacman package manager.

**Comment by ConnorDarghaoui** [comment]
Oh, now I feel very silly, I think that's it. if i talk about /mingw64. what should i do to "fix" this? .

**Comment by ZachBacon** [comment]
> Oh, now I feel very silly, I think that's it. if i talk about /mingw64. what should i do to "fix" this? 

pacman -S mingw-w64-x86_64-toolchain to get the most basic setup but if you want to see packages available go to https://packages.msys2.org to see what's there.

**Comment by aitesam961** [comment]
pacman -S --needed base-devel mingw-w64-x86_64-toolchain
Has to be executed in msys2.exe
Same happened to me. I actually ran the commands but in MinGW instead of msys2

**Comment by lazka** [comment]
(Ideally those empty dirs would be gone, but that is blocked by https://github.com/msys2/msys2-pacman/issues/17)

**Comment by Sharad-Bhatt** [comment]
i have the same problem as him  . When the installation is finished, no errors appear, but the "bin" and other folders are empty in my case it isn't running the command  pacman -S mingw-w64-ucrt-x86_64-gcc.
 just showing . -bash: $: command not found
 now what to do ?

**Comment by Sharad-Bhatt** [comment]
Guys i have a problem related to vs code after i have installed msys2 or previously installed mingw this problem was permanent thats why i have to change from mingw to msys2 but still nothing happened 
![image](https://github.com/msys2/msys2.github.io/assets/138997833/ffc6cbd9-48a7-4571-a5db-06fd9fb14e5a)
 this file do exists. i don't know what to do ?

**Comment by Biswa96** [comment]
@Sharad-Bhatt Please create a separate issue.

**Comment by ognevnydemon** [comment]
> Guys i have a problem related to vs code after i have installed msys2 or previously installed mingw this problem was permanent thats why i have to change from mingw to msys2 but still nothing happened 
> ![image](https://github.com/msys2/msys2.github.io/assets/138997833/ffc6cbd9-48a7-4571-a5db-06fd9fb14e5a)
>  this file do exists. i don't know what to do ?

your program is literally named `enter your program name, for example C:\Users\Uday\Desktop\Newcodesss\a.exe`

**Comment by Sharad-Bhatt** [comment]
it i have tried that to . but thanks i'll write a new issue separately

**Comment by AkuAdhan** [comment]
> pacman -S --needed base-devel mingw-w64-x86_64-toolchain Has to be executed in msys2.exe Same happened to me. I actually ran the commands but in MinGW instead of msys2

tysm broo...
